build/rollup.config.js

```diff
@@ -3,13 +3,15 @@ import nodeResolve from "@rollup/plugin-node-resolve";
 import modify from 'rollup-plugin-modify';
 import ascii from "rollup-plugin-ascii";
 import { terser } from "rollup-plugin-terser";
-import * as meta from "../package.json";
 import ignore from "rollup-plugin-ignore";
+import * as meta from "../package.json";
+
+const ignore_modules = ['fs', 'zlib'];
 
-const ingnore_modules = ['fs'];
+const importMetaUrlPolyfill = `(typeof document === 'undefined' && typeof location === 'undefined' ? undefined : typeof document === 'undefined' ? location.href : (document.currentScript && document.currentScript.src || new URL('jsroot.js', document.baseURI).href));`;
```

Comment:
The reason I had to define a custom polyfill for this is because the one from rollup includes `require` which can't be processed by webpack.

```js
(typeof document === 'undefined' && typeof location === 'undefined' ? new (require('u' + 'rl').URL)('file:' + __filename).href : typeof document === 'undefined' ? location.href : (document.currentScript && document.currentScript.src || new URL('jsroot.js', document.baseURI).href))
```

And this polyfill is applied at a time that the `rollup-plugin-ignore` is not able to ignore the `url` module.